tools/mtouch/mtouch.csproj

```diff
@@ -202,9 +202,6 @@
     <Compile Include="..\linker\ObjCExtensions.cs">
       <Link>Xamarin.Linker\ObjCExtensions.cs</Link>
     </Compile>
-    <Compile Include="..\linker\RemoveSelectors.cs">
```

Comment:
No (classic), but maybe that could change. Right now XM unified still use that pattern. As to why... I thought it was because XI had intrinsic for the call (but XM did not).